server/rpc/query/QueryHandler.java

```diff
@@ -90,8 +90,9 @@ public void handleRequest(Transaction.Req request) {
     }
 
     private void match(Transaction.Req request, QueryProto.Query.Match.Req req, Options.Query options) {
-        GraqlMatch query = Graql.parseQuery(req.getQuery()).asMatch();
-        ResourceIterator<ConceptMap> answers = queryManager.match(query, options);
+        if (!options.prefetch().isPresent()) options.prefetch(true);
+        final GraqlMatch query = Graql.parseQuery(req.getQuery()).asMatch();
+        final ResourceIterator<ConceptMap> answers = queryManager.match(query, options);
```

Comment:
Can we not bring back `final` for local variables?

Reply:
updated